laletterbuilder/letter_sending.py

```diff
@@ -62,10 +109,45 @@ def send_letter(letter: models.Letter):
             user=user,
         ).html
 
+    pdf_bytes = render_multilingual_letter(letter)
+    letter_type = letter.get_letter_type()  # TODO: localize this somewhere
+    ld = user.landlord_details
+
+    # TODO: fill in user pref
+    if ld.email:
+        email_letter_to_landlord(letter, pdf_bytes)
+
+    if ld.address_lines_for_mailing:
+        send_letter_via_lob(
+            letter,
+            pdf_bytes,
+            letter_description=f"{letter_type} letter",
+        )
+
+    if user.email:
+        pass
+        # TODO: add letter-email-to-user page on the front end
```

Comment:
Awesome. One optional way i've seen engineers keep track of TODOs in code is to put the ticket number (in our case, shortcut card) in the comment to make sure it's recorded somewhere that it's not done yet. Up to you if you want to start that convention here!